### PR TITLE
Feature/setup detekt

### PR DESCRIPTION
**Pull request by Uticodes**

### CODE REVIEW

File: app/src/test/java/com/example/universitieslist/ExampleUnitTest.kt
Comment (princessdharmy) [inline app/src/test/java/com/example/universitieslist/ExampleUnitTest.kt:1]: This also

File: detekt/detekt-config.yml
Comment (princessdharmy) [inline detekt/detekt-config.yml:1]: If you are not using the commented codes, you can take them out.

However, I did not see where composable functions are excluded from being documented. Or did you include it?

File: .gitignore
[the remark this answers is not in the document]
Comment (Uticodes) [reply .gitignore:1]: Yes, I discovered that .idea was not added and one of it's files was coming up when I added the detetk plugin, should i remove it for a new PR on that?

File: app/src/androidTest/java/com/example/universitieslist/ExampleInstrumentedTest.kt
[the remark this answers is not in the document]
Comment (Uticodes) [reply app/src/androidTest/java/com/example/universitieslist/ExampleInstrumentedTest.kt:1]: Detekt formatted it, I can't commit without the formatting, but I can omit the file, should I omit the file?

File: .gitignore
[the remark this answers is not in the document]
Comment (princessdharmy) [reply .gitignore:1]: It's okay. I could not figure out the changes. Hence, the reason I flagged it.

[the remark this answers is not in the document]
Comment (Uticodes) [reply .gitignore:1]: okay

File: detekt/detekt-config.yml
Comment (Uticodes) [reply detekt/detekt-config.yml:1]: Done

**Review by princessdharmy — APPROVED** [review]

**Pull request merged**